src/store/cache/contentNode/hooks.ts

```diff
@@ -76,12 +82,19 @@ const processNode = async (node: Node, aud: Audius): Promise<ContentNode> => {
       node.spID
     )
   }
+
+  const response = await fetch(`${node.endpoint}/health_check`)
+
+  const { data }: HealthCheckResponseData = await response.json()
+  const { country } = data
```

Comment:
I don't think this will work for discovery nodes, i don't think we actually even populate the `country` value, and we need the verbose health check to get lat/long

https://discoveryprovider.audius.co/health_check?verbose=true

maybe we need a server change?

Reply:
oh i see, was seeing the country flag in ui for discovery, but yeah looks like that was actually due to a default value. so for case of discovery providers, will need to get lat/long and use a library to spit out the country?

Reply:
or we could rely on lat/long and translate client side (for both discovery & content node). wdyt?